doc/tdshim_spec.md

```diff
@@ -504,6 +504,15 @@ In order to support AP wake up, TD Shim shall report [multiprocessor wakeup
 structure](#acpi-mpws) in MADT to share mailbox information with the payload or
 OS kernel, and send the OS commands via ACPI mailbox to wakeup APs.
 
+The page table for AP shall be in reserved memory or ACPI Nvs in memory map.
+The AP shall map the 4K OS wakeup page and the required memory for itself,
```

Comment:
What's the "4K OS wakeup page"? I could not find a reference for it.

Reply:
One page == 4K as minimal.
Other page size is 2M or 1G.

Reply:
Aha, got it. "The AP" means "the AP in td-shim mode".